clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -137,12 +139,14 @@ public String protocolType() {
 
     @Override
     public List<ProtocolMetadata> metadata() {
+        Set<String> subscribedTopics = new HashSet<>(subscriptions.subscription());
```

Comment:
Can `joinedSubscription` be initialized here (i.e. `this.joinedSubscription = new HashSet<>(subscriptions.subscription());`) and used instead of locally defined variable `subscribedTopics`?